assets/javascripts/discourse/controllers/s-show.js.es6

```diff
@@ -24,6 +25,17 @@ export default Controller.extend({
     bootbox.alert(I18n.t(`discourse_subscriptions.${path}`));
   },
 
+  @discourseComputed("model.product.repurchaseable", "model.product.subscribed")
+  canPurchase(repurchaseable, subscribed) {
```

Comment:
This isn't really dependent on `subscribed` at all, is it? I don't think you need this at all, and where `canPurchase` is used, you can instead just say `model.product.repurchaseable`

Reply:
oh nevermind if `!repurchaseable && !subscribed` it is true. Soooo this is a bit shorter 🤷 
```
 @discourseComputed("model.product.repurchaseable", "model.product.subscribed")
  canPurchase(repurchaseable, subscribed) {
    if (!repurchaseable && subscribed) {
      return false;
    }
    return true;
  }
  ```